backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -376,18 +381,18 @@ private RootCreatedBranchAndSkippedBranches createGitMacheteRootBranch(
       var createdRootBranch = new RootManagedBranchSnapshot(branchName, branchFullName,
           childBranches.getCreatedBranches(), pointedCommit, remoteTrackingBranch, syncToRemoteStatus, customAnnotation,
           statusHookOutput);
-      return RootCreatedBranchAndSkippedBranches.of(createdRootBranch, childBranches.getSkippedBranchNames());
+      return CreatedAndSkippedBranches.of(createdRootBranch, childBranches.getSkippedBranchNames());
```

Comment:
My first thought was that it could be a branch that is BOTH created and skipped :thinking: I assume that it is not so easy to reflect the accurate sense here

Reply:
My thinking may be the actual problem here :trollface:

Reply:
This could theoretically be `CreatedBranch{es,Snapshots}AndSkippedBranchNames`... but in turn this would be too verbose :/